src/Symfony/Component/Serializer/Tests/Normalizer/AbstractNormalizerTest.php

```diff
@@ -166,6 +167,14 @@ public function testObjectWithNullableNonOptionalConstructorArgumentWithoutInput
         $this->assertNull($dummy->getFoo());
     }
 
+    public function testObjectWithNullableNonOptionalConstructorArgumentWithoutInputAndRequireAllProperties()
+    {
+        $this->expectException(MissingConstructorArgumentsException::class);
```

Comment:
should be moved after L174

Reply:
I moved it down. But why this order? I couldn't find an example in the other tests for this order.

Reply:
the order ensures that only the expected line throws

Reply:
Ahh smart, didn't think of that.